Access Mgmt Login Rule and IDP doc updates (#23094)
gravitational/teleport: docs/pages/access-controls/login-rules.mdx

When users log in to your Teleport cluster with a configured SSO provider,
**Login Rules** can transform the traits provided by your IdP to meet your needs

> for configuring access within Teleport. Login Rules have been available starting

Reviewer: ```suggestion
for configuring access within Teleport. Login Rules are available starting
```
For me, "have been" implies that the docs belong to a specific time, i.e., like a news article/blog post does. I think the present is more appropriate here because (in theory, with the right environment) it's always possible to use Teleport v11.3.1.
